qiskit/chemistry/transformations/bosonic_transformation.py

```diff
@@ -237,7 +237,7 @@ def _map_bosonic_operator_to_qubit(bos_op: BosonicOperator,
     @staticmethod
     def _build_single_hopping_operator(index: List[List[int]],
                                        basis: List[int],
-                                       qubit_mapping: str) -> WeightedPauliOperator:
+                                       qubit_mapping: str) -> Union[PauliSumOp, None]:
```

Comment:
Optional[PauliSumOp] is equivalent to the Union of PauliSumOo and None and is what we normally use

Reply:
Thanks, I forgot it.